lib/matplotlib/colors.py

```diff
@@ -484,6 +484,21 @@ def makeMappingArray(N, data, gamma=1.0):
     return _create_lookup_table(N, data, gamma)
 
 
+def _deprecate_global_cmap(cmap):
```

Comment:
A bit more precise:
```suggestion
def _warn_if_global_cmap_modified(cmap):
```
The main purpose here is to warn. Technically, the deprecation has already happend.